CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+website/content/docs/contributors/community.md
```

Comment:
I created a symlink called `CONTRIBUTING.md` at root directory pointing to [website/content/docs/contributors/community.md](https://github.com/twitter/heron/blob/master/website/content/docs/contributors/community.md). I don't know why GitHub diff displays something like that.